src/android/CloudantSyncPlugin.java

```diff
@@ -509,29 +498,28 @@ public void run() {
 
     /**
      * Adds a single, possibly compound, index for the given field names
-     * @param datastoreName - The name of the Datastore
+     * @param documentStoreName - The name of the DocumentStore
      * @param fields - The list of fields to index
      * @param indexName - The name of the index
      * @param callbackContext - The javascript callback to execute when complete or errored
      */
-    private void ensureIndexed(final String datastoreName, final JSONArray fields, final String indexName, final CallbackContext callbackContext) {
+    private void ensureIndexed(final String documentStoreName, final JSONArray fields, final String indexName, final CallbackContext callbackContext) {
```

Comment:
Something for a follow-on PR but it appears we have no way of creating full text indexes which seems like a bit of an oversight